### PR TITLE
olm: created bundle for v3.6.0 (PROJQUAY-2276)

### PR DESCRIPTION
**Pull request by ricardomaraschini**
This PR creates a directory called `bundle` from where the OLM release manifests must be copied during the release process.

### CODE REVIEW

**Comment by ricardomaraschini** [comment]
@jonathankingfc @harishsurf @thomasmckay Not sure who can review this but flagging you three.

File: bundle/manifests/quay-bridge-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
...Gj6luGDbRAHKTdzCL52XpgjCZu3GJonQnCjW1jcI1/zgmVZaKqrqkuW1KcSy8pljkxNBsMmg+4BMrwKVepwJQMcltavXIJkZE6574exgr7yk9tJp0R0tTZUhCnckLfzafU0QhD3aR22qXQSdzl0mnBQyrWdvbuihZT+0OE6F1evvk3JJQmNzaBAxIOt10LIgCMPoaUCUmh9ULGDJuMQrhKj2JTIuMVWAclIS8x5AeQrCXKL2CQZA6RCZ/RluUgH7QPxySbR0VMclQZgqQHESFvq83AE/QRhGXMJzUzqdbqrhXt9w4uUOjo5LPlESF64jInckLTwrXGT4CsJGXJ62ksW929VUp0uzZS93cMglWcUsQYPPfnfTs3KadwQdPe4rCGMfewCWMl5nFJrqdGVWpl896PuePSkrKxNc4h40hIWOHU9Y2BH0YwIFYbN3sXpUBC8xOGE7Xdlc0pWDzaFdEhQWdVgQw3POWbkXgYJgQpQW9jQyKYLEgGanm7r75hBYNtOaHejTcgnPEvgNGnQpsdC+qcf+QbF/rnL2e9EZvOxRqqou4eFENdMaxiWmy7MJS+60JSlBWOhqCupPVM61pb5E54Mq/eCCUqaVXKK6R4TOTqnKjhU2f5qA+o1SCw8VC1UcIw90MXnI8O1GWQxf3dRj/0b2bSkttmZD4W84r82L4Q89h0pFDOisfmez+IIo82L4M20PBQMTZTiP5+bF8EZXDIR9Fi6dzExPIxMa5jBhxEDYDTv0i+kFCA1zlLBiwMQOKnoB9Gg4q3BUx9yEPYltf1gxYPLx3W/VWFvpaWT8ZLS5Ak362DxDfS2SB8b2qdMLKrVwN6UIhMbbFHqv9J5NiQGTDnFCTyOjB2DZBTffflCIokShTG5KlUgEwUQIS9EDsOhsc6HxFobqGZRCl02FqBKZIBw62JGeuUSP+REabyGo7EqVvqDikgkiF4RDcxZ6zA89WUZonMXQ6hBakGBiBCXDtAnCIWHoyTKzfTRGoydaNzVdQnCmXRAOhTJ6mMls62Ooj6DlnVGHJj9mTBAOdf70/Ax6ZAM9JUC4YRqg/Rm0JYBWoUfVWcsy44I4IXHoKQF0MD2dhR5VWKNwRHv6aBsZ7VyaaRGczCpBvKDQRic+05m29EVHqcoKRR88O66CNuR30T7wmQpFUgD4Px6QRGRh7pGzAAAAAElFTkSuQmCC
```
Comment (thomasmckay) [inline bundle/manifests/quay-bridge-operator.clusterserviceversion.yaml:109]: Is this the project quay (blue) or the red hat (red) icon?

Comment (ricardomaraschini) [reply bundle/manifests/quay-bridge-operator.clusterserviceversion.yaml:109]: This one:
image

**Comment by ricardomaraschini** [comment]
@jonathankingfc @harishsurf @thomasmckay This is still pending a review, could any of you do it?

**Review by jonathankingfc — APPROVED** [review]

**Comment by ricardomaraschini** [comment]
Seems like I don't have enough permissions to merge this.

**Comment by BillDett** [comment]
Try merging again @ricardomaraschini - perms were set wrong on this repo.

**Pull request merged**